Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10
+FROM python:3.11
```

Comment:
```suggestion
FROM python:3.12
```
Just wondering: if we merge #41 first, this should probably become 3.12 too, right?

Reply:
I guess we can try!